ledger/participant-integration-api/src/main/scala/platform/store/ActiveLedgerStateManager.scala

```diff
@@ -53,7 +76,7 @@ private[platform] class ActiveLedgerStateManager[ALS <: ActiveLedgerState[ALS]](
 
   private object AddTransactionState {
     def apply(acs: ALS): AddTransactionState =
-      AddTransactionState(Some(acs), Set(), Set.empty, Set.empty)
+      AddTransactionState(RollbackState(Set.empty, Set.empty, Some(acs)), List(), Set(), Set.empty)
```

Comment:
What's the difference between `Set()` and `Set.empty`?

Reply:
None, the former might be a tiny bit slower. Changed to `Set.empty` in ebe77bbc90c77749e29a9fad6b7f0077fc3c0713 (I just kept what was there before).

Reply:
Thanks for the explanation! :)